Sources/Common/Core/ScalarsToColors/index.js

```diff
@@ -125,25 +125,26 @@ function vtkScalarsToColors(publicAPI, model) {
       newColors[count] = l;
       newColors[count + 1] = l;
       newColors[count + 2] = l;
-      newColors[count + 3] = convtFun(values[i + 1]);
+      newColors[count + 3] = convtFun(values[i + 1]) * alpha;
```

Comment:
I think it is correct. The code in VTK is
      outPtr[3] = static_cast<unsigned char>(a*alpha + 0.5);
which has the multiplication